incubator/cloudbeaver/values.yaml

```diff
@@ -0,0 +1,121 @@
+init:
+  #This image is used by the 2nd init container which relies on the jar binary to inject the xml files
+  image:
+    registry: docker.io
+    repository: openjdk
+    tag: 11-jdk
+    pullPolicy: IfNotPresent
+
+cloudbeaver:
+  #Cloudbeaver image details
+  image:
+    registry: docker.io
+    repository: dbeaver/cloudbeaver
+    tag: latest
+    pullPolicy: IfNotPresent
+
+  #Override dynamic name creation
+  #nameOverride: 
+
+  #Set the driver version to use.
+  #If download is set to false, the nuodb jar is expected to exist in the files directory, then it will be loaded as a configmap.
+  #Get it from https://mvnrepository.com/artifact/com.nuodb.jdbc/nuodb-jdbc) and load it with:
+  #kubectl -n cloudbeaver create configmap kubectl create configmap nuodb-jdbc-jar --from-file=nuodb-jdbc-23.0.0.jar=nuodb-jdbc-23.0.0.jar
+  nuodbDriver:
+    download: true
+    version: 23.0.0
+    
+  #These values populate the files in files/cloudbeaver/*
+  #Refer to this page for information on what these do: https://github.com/dbeaver/cloudbeaver/wiki/Server-configuration
+  configs:
+
+    #Optionally the overrides can be disbaled, instead using all cloudbeaver defaults.
+    override: true
+
+    #If any of these sections are not set, Cloudbeaver will take you through the setup on the first start.
+    serverName: Cloudbeaver
+    serverURL: http://localhost:8978
```

Comment:
Should the URL be changed if a loadbalancer service or different port is used?

Reply:
As discussed, removed this from being configurable. As per Cloudbeaver docs it is only used for 3rd party integration links.